modules/openapi-generator/src/main/resources/go/model_simple.mustache

```diff
@@ -167,6 +167,18 @@ func (o *{{classname}}) Set{{name}}(v {{vendorExtensions.x-go-base-type}}) {
 	o.{{name}} = v
 {{/isNullable}}
 }
+{{#useDefaultValuesForRequiredVars}}
+{{^isReadOnly}}
+{{#defaultValue}}
+// GetDefault{{baseName}} function assigns the default value {{defaultValue}} to the {{name}} field
+// of the {{classname}} struct and returns the {{{defaultValue}}}.
+func (o *{{classname}}) GetDefault{{baseName}}() interface{}  {
```

Comment:
instead of using `baseName`, what about using `nameInPascalCase` instead to conform to Go method naming convention?

Reply:
Fixed